### PR TITLE
Update kubectl Download Options

### PR DESCRIPTION
**Pull request by network-charles**
### Description

This PR clearly describes the two options for installing kubectl on Windows.

### Issue

Closes: #47614

### CODE REVIEW

File: content/en/docs/tasks/tools/install-kubectl-windows.md [hunk 1]
```diff
@@ -18,19 +18,24 @@ Using the latest compatible version of kubectl helps avoid unforeseen issues.
...
+   - Using curl:
 
-   ```powershell
-   curl.exe -LO "https://dl.k8s.io/release/v{{< skew currentPatchVersion >}}/bin/windows/amd64/kubectl.exe"
-   ```
+     If you have `curl` installed, use this command:
+
+     ```powershell
+     curl.exe -LO "https://dl.k8s.io/release/v{{< skew currentPatchVersion >}}/bin/windows/amd64/kubectl.exe"
+     ```
```
Comment (sftim) [inline content/en/docs/tasks/tools/install-kubectl-windows.md:38]: (nit) Could also use [tabs](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#tabs) here.

**Review by sftim — COMMENTED** [review]
Thanks. This change makes the site better.

/approve

File: content/en/docs/tasks/tools/install-kubectl-windows.md (hunk 1 above)
Comment (network-charles) [reply content/en/docs/tasks/tools/install-kubectl-windows.md:38]: Ah, I didn’t know about this feature. I’ll consider that next time, the PR has been merged.

**Pull request merged**